Jenkinsfile

```diff
@@ -236,6 +236,17 @@ stage('Integration Test') {
       }
     }
   },
+  'frontend: GPU': {
```

Comment:
@tqchen please check the Jenkinsfile change to add relay frontend tests. Do we have to merge it so that it can take effect?

Should we put all relay frontend tests in tests/python/frontend? I see mxnet tests in both tests/python/frontend and tests/python/relay/frontend